### PR TITLE
Handle potentially empty but present started_at property of pod state

### PR DESCRIPTION
**Pull request by jfongatyelp**
This should fix the occasional 500s we get when a pod has recently started and has a container state with an empty `started_at` property.  
```
2022-03-25 12:26:53 | ERROR:paasta_tools.api.views.exception:Traceback (most recent call last):
2022-03-25 12:26:53 | AttributeError: 'NoneType' object has no attribute 'timestamp'
2022-03-25 12:26:53 | start_timestamp = this_state["started_at"].timestamp()
2022-03-25 12:26:53 | File "/opt/venvs/paasta-tools/lib/python3.7/site-packages/paasta_tools/instance/kubernetes.py", line 830, in get_pod_containers
2022-03-25 12:26:53 | "containers": containers_task.result(),
2022-03-25 12:26:53 | File "/opt/venvs/paasta-tools/lib/python3.7/site-packages/paasta_tools/instance/kubernetes.py", line 776, in get_pod_status
2022-03-25 12:26:53 | "pods": await asyncio.gather(*pod_status_tasks) if pod_status_tasks else [],
2022-03-25 12:26:53

<br class="Apple-interchange-newline">
```

We had correctly handled this when looking at `last_state` for the container, but not the current one. 

This also adds another test for `get_pod_containers` to check this exact issue sets `timestamp` correctly w/o throwing errors (None)

### CODE REVIEW

File: paasta_tools/instance/kubernetes.py [hunk 1]
```diff
@@ -826,7 +826,7 @@ async def get_pod_containers(
                     reason = this_state["reason"]
                 if "message" in this_state:
                     message = this_state["message"]
-                if "started_at" in this_state:
+                if this_state.get("started_at"):
                     start_timestamp = this_state["started_at"].timestamp()
```
Comment (kawaiwanyelp) [inline paasta_tools/instance/kubernetes.py:830]: If `started_at` is `None`, then there would be an exception when calling `.timestamp()` I think. You'll also need to check for the non-null case, but I'm not sure what you would retrieve here. Just set `this_state["started_at"]` to `None` maybe?

**Review by kawaiwanyelp — CHANGES_REQUESTED** [review]

File: paasta_tools/instance/kubernetes.py (hunk 1 above)
Comment (jfongatyelp) [reply paasta_tools/instance/kubernetes.py:830]: if started_at is None or empty or otherwise False-y, we won't get to the point where we call `this_state["started_at"].timestamp()`. That's exactly the situation we are trying to fix here.  

I figured we don't really need to distinguish from the state not having this info at all to having unuseful/blank info.

**Review by kawaiwanyelp — APPROVED** [review]

**Review by lanceleih — APPROVED** [review]
LGTM
